pokete_classes/hotkeys.py

```diff
@@ -130,10 +136,13 @@ def get_X_strength(self) -> int:
     'p': ActionList([Action.POKEDEX]),
     'f': ActionList([Action.FREE_POKETE]),
     'm': ActionList([Action.MAP, Action.MOVE_POKETE]),
-    'c': ActionList([Action.CLOCK]),
+    'c': ActionList([Action.CLOCK, Action.QUICK_ATC_3]),
     '?': ActionList([Action.HELP, Action.INFO]),
     'e': ActionList([Action.MENU]),
     ':': ActionList([Action.CONSOLE]),
+
```

Comment:
For QWERTY keyboard layouts.

```suggestion

    'z': ActionList([Action.QUICK_ATC_1]),
```